trigger/poll/watcher.go

```diff
@@ -263,7 +265,7 @@ func (w *RepositoryWatcher) addJob(ti *types.TrackedImage, schedule string) erro
 	// and for non-semver types we create a single tag watcher which
 	// checks digest
 	_, err = version.GetVersion(ti.Image.Tag())
-	if err != nil {
+	if err != nil || (ti.Policy != nil && ti.Policy.Name() == "force") {
```

Comment:
could there be some explanation here? (as it doesn't go with what the comment above the Getversion() is). For example 
`ti.Policy != nil && ti.Policy.Name() == "force"` is used in this functon twice for the `keepTag`. Could there be some nicer way?

Reply:
Good point. It might be a _left-over_ of my _go back and forth_ testing.
And comment should indeed reflect the behaviour below it.

Reply:
About the nicer way, I wanted to perform the _minimum_ change.
The other _issue_ I fall in, is that, within `Watch` I could have used `TrackedImage` type to find `Policy` to make a cleaner call to `getImageIdentifier`, but in `Unwatch`, we are _only_ provided with a string representing the version, but no way to find its `Policy`.

Any advice on how to do it are welcome, though.